image_filtering_node:
  ros__parameters:
    image_topic: "/flir_camera/image_raw"
    filter_params:
      filter_type: "ebus"
      unsharpening:
        blur_size: 8
      erosion:
        size: 1
      dilation:
        size: 1
      white_balancing:
        contrast_percentage: 0.8
      ebus:
        erosion_size: 2
        blur_size: 30
        mask_weight: 5

# Filter params should reflect the FilterParams struct 
# defined in /include/image_filters/image_processing.hpp